system-int-test/test/cmr/system_int_test/ingest/collection_ingest_test.clj

```diff
@@ -55,6 +55,27 @@
       (is (= 5 revision-id))
       (is (mdb/concept-exists-in-mdb? concept-id 5)))))
 
+;; Verify a concept can be ingested twice to get two revisions and ignore_conflict can impact the reindex status. 
+(deftest collection-ingest-test
+  (testing "ingest of a new concept twice to get two revisions, and reindex revision 1, with ignore_conflict on and off."
+    (let [concept (data-umm-c/collection-concept {})
+          ingested-concept1 (ingest/ingest-concept concept)
+          ingested-concept2 (ingest/ingest-concept concept)
+          concept-id1 (:concept-id ingested-concept1)
+          concept-id2 (:concept-id ingested-concept2)
+          revision-id1 (:revision-id ingested-concept1)
+          revision-id2 (:revision-id ingested-concept2)]
+      (index/wait-until-indexed)
+      (is (mdb/concept-exists-in-mdb? concept-id1 revision-id1))
+      (is (mdb/concept-exists-in-mdb? concept-id2 revision-id2))
+      (is (= 1 revision-id1))
+      (is (= 2 revision-id2))
+      (is (= concept-id1 concept-id2))
+      (index/reindex-concept-ignore-conflict-default concept-id1 revision-id1)
+      (index/reindex-concept-ignore-conflict-true concept-id1 revision-id1)
+      (index/reindex-concept-ignore-conflict-false concept-id1 revision-id1)
```

Comment:
As Chris has pointed out, the reindex status check should happen here. The `util` namespace would only have a reindex function that takes the concept-id, revision-id and ignore-conflict params.

Reply:
I was thinking of the same thing actually. it's much more readable. Thanks.